docs/concepts/what-is-a-claimmapper-plugin.md

```diff
@@ -0,0 +1,120 @@
+---
+id: what-is-a-claimmapper-plugin
+title: What is a ClaimMapper Plugin?
+sidebar_label: What is a ClaimMapper Plugin?
+---
+
+`ClaimMapper` is a plugin that extracts claims from JSON Web Tokens (JWT).
+This process is achieved with the method `GetClaims`, which translates `AuthInfo` structs from the caller into `Claims` about the caller's roles within Temporal.
+
+A `Role` (within Temporal) is a bit mask that combines one or more of the role constants.
+In the example below, the role is assigned constants that allow the caller to read and write information.
+
+```go
+role := authorization.RoleReader | authorization.RoleWriter
+```
+
+`GetClaims` is customizable, and can be modified with the `temporal.WithClaimMapper` server option.
+Temporal also offers a default JWT `ClaimMapper` for your use.
+
+A typical approach is for `ClaimMapper` to interpret custom `Claims` from a caller's JSON Web Token (JWT), such as membership in groups, and map them to Temporal roles for the user.
+The subject information from the caller's TLS certificate can also be a parameter in determining roles.
+
+### `AuthInfo`
+
+`AuthInfo` is a struct that is passed to `GetClaims`. `AuthInfo` contains an authorization token extracted from the `authorization` header of the gRPC request.
+
+`AuthInfo` includes a pointer to the `pkix.Name` struct.
+This struct contains an [x.509](https://www.ibm.com/docs/en/ibm-mq/7.5?topic=certificates-distinguished-names) distinguishable name from the caller's mTLS certificate.
+
+### `Claims`
+
+`Claims` is a struct that contains information about permission claims granted to the caller.
+
+`Authorizer` assumes that the caller has been properly [authenticated](/docs/security/authentication), and trusts the `Claims` when making an authorization decision.
```

Comment:
This is a broken link - 
Does it need to be linked at all? Where would it go if it were?